src/improvements/tasks/MultisigTask.sol

```diff
@@ -630,11 +674,16 @@ abstract contract MultisigTask is Test, Script {
     function printNestedHashToApprove() public view {
         bytes memory callData = generateApproveMulticallData();
 
-        if (childMultisig != address(0)) {
+        if (childMultisig != address(0) && childMultisig.code.length != 0) {
```

Comment:
Similar comments for this method. I would just create a single `printNestedData()` method to hold all of:
```solidity
if (isNestedSafe) {
    console.log("\n\n------------------ Nested Multisig EOAs Data to Sign ------------------");
    printNestedDataToSign();
    console.log("\n\n------------------ Nested Multisig EOAs Hash to Approve ------------------");
    printNestedHashToApprove();
```
that logic. Because both `printNestedDataToSign` and `printNestedHashToApprove` are only called once anyway, so this way we can clearly document all the logs and logic for the nested case.

`printDataToSign` and `printHashToApprove` are also both only used once, so we can again bundle the logs and logic for the single case in the `else` block